testing/ostest/ostest.h

```diff
@@ -27,6 +27,8 @@
 
 #include <nuttx/config.h>
 
+#include <assert.h>
```

Comment:
In general I do not support indirect including. What from ostest.h needs assert.h?

Reply:
Agreed, `assert.h` should be included directly in the files where it is required.